frontend_build/src/webpack.config.base.js

```diff
@@ -77,9 +77,9 @@ var config = {
   ],
   resolve: {
     alias: {
-      'kolibri_module': path.join(__dirname, '..', '..', 'kolibri', 'core', 'assets', 'src', 'kolibri_module'),
-      'core-base': path.join(__dirname, '..', '..', 'kolibri', 'core', 'assets', 'src', 'core-base'),
-      'core-theme.styl': path.join(__dirname, '..', '..', 'kolibri', 'core', 'assets', 'src', 'core-theme.styl'),
+      'kolibri_module': path.resolve('kolibri/core/assets/src/kolibri_module'),
+      'core-base': path.resolve('kolibri/core/assets/src/core-base'),
+      'core-theme.styl': path.resolve('kolibri/core/assets/src/core-theme.styl'),
```

Comment:
That is my understanding, corroborated here: http://shapeshed.com/writing-cross-platform-node/